src/it/scala/org/zalando/spearheads/innkeeper/routes/PostRoutesHostSpec.scala

```diff
@@ -39,6 +39,17 @@ class PostRoutesHostSpec extends FunSpec with BeforeAndAfter with Matchers {
           route.createdBy should be(UserName("user~1"))
           route.route.matcher.hostMatcher.get should be("www.yahoo.com")
         }
+
+        it("should not create more routes") {
+          val routeName = "route_regex_1"
```

Comment:
Do you really need to have it as a separate variable? It is used only in one place. Can be inlined. If needed then named parameters can be used.